Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```diff
@@ -867,6 +867,7 @@
   metamorphicChangeColor: false
   metabolisms:
     Drink:
+      metabolismRate: 0.5
```

Comment:
```suggestion
```
Isn't this the default rate? So it can be removed I think.

Reply:
Good call.